### PR TITLE
Persist and expose PayPal Payout errors

### PR DESCRIPTION
**Pull request by kewitz**
Resolves https://github.com/opencollective/opencollective/issues/3251

### CODE REVIEW

File: server/paymentProviders/paypal/payouts.ts [hunk 1]
```diff
@@ -50,16 +51,25 @@ export const payExpensesBatch = async (expenses: any[]): Promise<any[]> => {
...
-  });
-  return Promise.all(updateExpenses);
+  try {
+    const response = await paypal.executePayouts(connectedAccount, requestBody);
+    const updateExpenses = expenses.map(async e => {
+      await e.update({ data: response.batch_header, status: status.PROCESSING });
+      const user = await models.User.findByPk(e.lastEditedById);
+      await e.createActivity(activities.COLLECTIVE_EXPENSE_PROCESSING, user);
+    });
+    return Promise.all(updateExpenses);
```
Comment (Betree) [inline server/paymentProviders/paypal/payouts.ts:61]: Is is possible to have only one expense in the batch failing? Or do the entire batch fails if just one fails?

File: server/graphql/v2/object/Activity.ts [hunk 2]
```diff
@@ -52,11 +52,11 @@ export const Activity = new GraphQLObjectType({
       type: new GraphQLNonNull(GraphQLJSON),
       description: 'Data attached to this activity (if any)',
       resolve(activity): object {
+        const toPick = ['error'];
```
Comment (Betree) [inline server/graphql/v2/object/Activity.ts:55]: Are we 100% that the error message can't contain sensitive or private info, such as accounts IDs or tokens?

Comment (Betree) [reply server/graphql/v2/object/Activity.ts:55]: If we're not sure, I think we should pick this field only if `remoteUser` is a host admin.

Comment (kewitz) [reply server/graphql/v2/object/Activity.ts:55]: Well, this is out of our control so there's no way to be certain. I'll validate `remoteUser` is host admin.

File: server/paymentProviders/paypal/payouts.ts (hunk 1 above)
Comment (kewitz) [reply server/paymentProviders/paypal/payouts.ts:61]: That is a really nice question! I was taking for granted that everything fails.
I just tested to make sure that was the behavior and it is, the whole batch fails.

**Pull request merged**